website/content/docs/configure-hcp-packer.mdx

```diff
@@ -0,0 +1,53 @@
+---
+description: |
+  You can configure Packer to publish metadata for all completed builds to an active HCP Packer registry. 
+  Legacy JSON templates can connect to the registry using environment variables. HCL2 templates can connect using an hcp_packer_registry block. 
+page_title: Configuring HCP Packer
+---
+
+# Configuring HCP Packer
+
+[HCP Packer](https://cloud.hashicorp.com/docs/packer) stores metadata about your Packer images so that you can track updates, 
+use the most up-to-date base images, and deploy the most up-to-date downstream images. 
+
+HCP Packer can be used with both JSON and HCL2 templates. If you are using JSON templates we recommend getting started with 
+the [HCP Packer environment variables](#hcp-packer-environment-variables) and then, when ready, migrate from JSON to HCL2 templates. 
+
+~> Refer to the [HCP Packer Template Configuration](https://cloud.hashicorp.com/docs/packer/store-image-metadata/template-configuration) documentation for details 
+on using HCL2 templates to push image metadata to an active HCP Packer registry.
+
+## HCP Packer Environment Variables
+
+The following environment variables allow you to configure Packer to push image metadata to an active registry without changing your template. 
+
+Environment variables are the only way to configure HCP Packer from a JSON template. 
+
+You must set following environment variables to enable Packer to push metadata to a registry.
```

Comment:
```suggestion
The following environment variables let you configure Packer to push image metadata to an active registry without changing your template. You can use environment variables with both JSON and HCL2 templates. Refer to [Basic Configuration With Environment Variables](https://cloud.hashicorp.com/docs/packer/store-image-metadata/template-configuration#basic-configuration-with-environment-variables) in the HCP Packer documentation for complete instructions and examples. 

You must set following environment variables to enable Packer to push metadata to a registry.
```

Reply:
I like the idea of having the Basic Configuration section.